server/apps/newsletter/tasks.py

```
@@ -51,17 +53,23 @@ def send_newsletter_via_email():
+        response = beehiiv.create_subscriber(email, is_active=True)
+        logger.info(f"Beehiiv API response for {email}: {response}")
     except ValueError as e:
         # Handle Beehiiv "invalid" status
         logger.warning(f"Beehiiv sync warning for {email}: {str(e)}")
     except Exception as e:
-        # Log the error
-        logger.error(f"Error syncing {email} to Beehiiv: {str(e)}")
\ No newline at end of file
+        # Log the error and retry
+        logger.error(f"Error syncing {email} to Beehiiv: {str(e)}")
+        raise self.retry(exc=e, countdown=60)  # Retry after 60 seconds
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)